test/use-package-test.el

```diff
@@ -1,4 +1,4 @@
-;;; use-package-tests.el --- Tests for use-package.el
+;;; use-package-test.el --- Tests for use-package.el
```

Comment:
I named it with the `-test.el` suffix because that was the exact filename generated by ert-runner's init feature. It might work without this rename but I haven't tried it yet (and the file would still have to be in the `test` directory for ert-runner). Also I thought `-test.el` was the convention but I could be wrong.